serveradmin/dataset/filters.py

```diff
@@ -515,13 +480,23 @@ def from_obj(cls, obj):
 
 class InsideNetwork(Overlap):
     def as_sql_expr(self, attribute, servertypes):
+        if attribute.type != 'inet':
+            raise FilterValueError(
+                'Cannot network filter attribute "{}"'.format(attribute)
+            )
+
         return _condition_sql(attribute, "{{0}} <<= ANY('{{{{{0}}}}}')".format(
             ','.join(str(n) for n in self.networks)
         ), servertypes)
 
 
 class InsideOnlyNetwork(InsideNetwork):
     def as_sql_expr(self, attribute, servertypes):
+        if attribute.type != 'inet':
```

Comment:
I will clean it up again with the next one.  One of the purposes of this commit is to let the server-side filters go way completely.  Then we would have them all in a single place.